code/game/objects/effects/spawners/windowspawner.dm

```diff
@@ -2,12 +2,12 @@
 	name = "window spawner"
 	icon = 'icons/obj/structures.dmi'
 	icon_state = "window_spawner"
-	var/useFull = 0
-	var/useGrille = 1
-	var/windowtospawn = /obj/structure/window/basic
-	anchored = 1 // No sliding out while you prime
+	var/useFull = TRUE
```

Comment:
Since `windowtospawn` is a full window either way, setting this to `FALSE` would potentially lead to multiple full windows being created on the same turf. I can see two such cases happening by searching for that variable on the repository.

Reply:
Yeahp---should be handled now.